src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -79,4 +81,20 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  /**
+   *Gets list of all users
```

Comment:
Nit: there should be a space after the * for docstrings. See: https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line